docs/spec/channel.md

```diff
@@ -146,39 +146,91 @@ Each channel is _namespaced_ and MUST have the following:
 - label of `duck.knative.dev/addressable: "true"`
 - The category `channel`
 
+#### Label Requirements
+
+Each instantiated Channel (ie, CO) must have a label indicating which version of the
```

Comment:
Change `must` to `MUST`

Reply:
Oooh, good. Since we assume that omission means v1alpha1, I changed to SHOULD? Ok?

Reply:
can we avoid abbreviations? CO -> Custom Object